py-polars/polars/utils.py

```diff
@@ -197,6 +197,7 @@ def _prepare_row_count_args(
 
 
 EPOCH = datetime(1970, 1, 1).replace(tzinfo=None)
+UTC = zoneinfo.ZoneInfo("UTC")
```

Comment:
FYI @ritchie46 - this breaks imports of Polars in Google Colab (Python 3.7) if installed as `polars` without `polars[timezone]`:

```
!pip install polars

import polars
```

```
[/usr/local/lib/python3.7/dist-packages/polars/utils.py](https://localhost:8080/#) in <module>
    198 
    199 EPOCH = datetime(1970, 1, 1).replace(tzinfo=None)
--> 200 UTC = zoneinfo.ZoneInfo("UTC")
    201 
    202 

NameError: name 'zoneinfo' is not defined
```

Is it possible to handle the `_ZONEINFO_AVAILABLE=False` case a little more elegantly, and to specify `backports.timezone` as a dependency if the Python version is <3.9?

Reply:
***xref:*** #5244

Reply:
Ai.. Will patch this today